greenplumpython/table.py

```diff
@@ -96,6 +96,22 @@ def top(self, count: int, order_by: Iterable, skip: int = 0):
             parents=[self],
         )
 
+    def union(
+        self,
+        other: "Table",
+        all: bool = False,
```

Comment:
is_all maybe better?
all somehow is a function

Reply:
Changed `is_all` to `all` to keep it consistent with the signature of `fetch()`, maybe change both to `is_all` will be better? @xuebinsu